ocaml/xapi/xapi_ha.ml

```diff
@@ -558,6 +558,7 @@ module Monitor = struct
                 end
               end
             with e ->
+              log_backtrace ();
```

Comment:
Would this be better under the below debug line? Maybe that would look clearer in the logs.

Reply:
I'm afraid we'll loose the backtrace then (`Format` sometimes raises/catches some internal exceptions)